tests/testthat/test-epi_slide.R

```diff
@@ -361,7 +361,7 @@ test_that("computation output formats x as_list_col", {
         value,
         before = 6L, as_list_col = TRUE, na.rm = TRUE
       ),
-    class = "epiproces__epi_slide_mean__list_not_supported"
+    class = "epiprocess__epi_slide_opt__list_not_supported"
```

Comment:
(suggest: up at line 317 (or 331, in my editor??), rename the test ("computation output formats x as_list_col") or split into multiple so it makes sense)